### PR TITLE
Add Launch Pad keymaps

### PR DESCRIPTION
**Pull request by luis-godinez**
Various updates for Ungodly Design Launch Pad.

## Description

- Update readme. HTML tables were used, in lieu of markdown, to ensure table columns render as intended.
- New keymaps:
  - **Default**: User requested layout.
  - **Warzone**: User requested layout.
  - **VIA**: Adds via configurator functionality. Confirmed to be working with midi slider.
- Force NumLock On when keyboard is initialized.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ungodly/launch_pad/readme.md [hunk 1]
```diff
@@ -35,3 +33,63 @@ Make example for this keyboard (after setting up your build environment):
...
+
+Layer | Graphic
+:----:|:----:
+home  |  ![home](https://i.imgur.com/tK3u7ZU.png)
+nav  |  ![nav](https://i.imgur.com/J0FNZfR.png)
+rgb  |  ![rgb](https://i.imgur.com/XK6r6KL.png)
+
+* These instructions require compiling your own firmware. [Ensure that your build environenment is setup](https://beta.docs.qmk.fm/tutorial/newbs_getting_started).
+* Create 128x32 px black and white pixel art image. The graphic will be rotated 90 degrees clockwise and rendered vertically on the oled display.
+* Load the image on [Image2cpp](https://javl.github.io/image2cpp/)
```
Comment (drashna) [inline keyboards/ungodly/launch_pad/readme.md:86]: https://joric.github.io/qle/ may be a better way to edit and load images

**Review by drashna — APPROVED** [review]

File: keyboards/ungodly/launch_pad/readme.md (hunk 1 above)
Comment (luis-godinez) [reply keyboards/ungodly/launch_pad/readme.md:86]: Thanks! I updated the readme.

**Comment by luis-godinez** [comment]
Anything I can do to move this forward?

**Comment by drashna** [comment]
This has merge conflicts that need to be resolved.

**Comment by luis-godinez** [comment]
bump

File: keyboards/ungodly/launch_pad/keymaps/default/keymap.c [hunk 2]
```diff
@@ -17,10 +17,55 @@
 #include "analog.h"
 #include "qmk_midi.h"
 
+/* Force Numlock on */
+void matrix_init_user (void) {
+  if (!(host_keyboard_leds() & (1<<USB_LED_NUM_LOCK))) {
+      register_code(KC_NUMLOCK);
+      unregister_code(KC_NUMLOCK);
```
Comment (fauxpark) [inline keyboards/ungodly/launch_pad/keymaps/default/keymap.c:24]:
```suggestion
  if (!host_keyboard_led_state().num_lock) {
      tap_code(KC_NUMLOCK);
```

File: keyboards/ungodly/launch_pad/keymaps/via/keymap.c [hunk 3]
```diff
@@ -17,10 +17,55 @@
 #include "analog.h"
 #include "qmk_midi.h"
 
+/* Force Numlock on */
+void matrix_init_user (void) {
+  if (!(host_keyboard_leds() & (1<<USB_LED_NUM_LOCK))) {
+      register_code(KC_NUMLOCK);
+      unregister_code(KC_NUMLOCK);
```
Comment (fauxpark) [inline keyboards/ungodly/launch_pad/keymaps/via/keymap.c:24]:
```suggestion
  if (!host_keyboard_led_state().num_lock) {
      tap_code(KC_NUMLOCK);
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ungodly/launch_pad/rules.mk [hunk 4]
```diff
@@ -2,7 +2,7 @@
 MCU = atmega32u4
 
 # Bootloader selection
-BOOTLOADER = atmel-dfu
+BOOTLOADER = qmk-dfu
```
Comment (fauxpark) [inline keyboards/ungodly/launch_pad/rules.mk:5]: If you really are using qmk-dfu, you should have the corresponding defines in the config.h:
https://docs.qmk.fm/#/flashing?id=qmk-dfu

Comment (luis-godinez) [reply keyboards/ungodly/launch_pad/rules.mk]: I'll roll it back. Only using it for production hex.

**Review by fauxpark — APPROVED** [review]

**Comment by luis-godinez** [comment]
Is this pending another reviewer or can it be merged?

**Comment by drashna** [comment]
Thanks!

**Pull request merged**